src/oasis-indexer/generated/api.ts

```diff
@@ -565,14 +565,14 @@ detected or is not supported, this field will be null/absent.
   /** The number of addresses that have a nonzero balance of this token,
 as calculated from Transfer events.
  */
-  num_holders: number;
+  num_holders: bigint;
 }
 
 export interface RuntimeStatus {
   /** The number of compute nodes that are registered and can run the runtime. */
   active_nodes: number;
   /** The height of the most recent indexed block (also sometimes referred to as "round") for this runtime. Query a synced Oasis node for the latest block produced. */
-  latest_block: number;
+  latest_block: bigint;
```

Comment:
This is odd. I'll have to investigate upstream

Reply:
> I'm suspicious of this change. [anymaniax/orval#824 (comment)](https://github.com/anymaniax/orval/issues/824#issuecomment-1539187873)
> 
> If we upgrade orval, it is also reverted, and made opt-in

OK, upgrading orval in https://github.com/oasisprotocol/explorer/pull/421.